config/initializers/0_clowder-config.rb

```diff
@@ -1,19 +1,29 @@
+# frozen_string_literal: true
+
+def build_endpoint_url(endpoint, config)
+  # There is currently no tlsPort or anything similar in clowder-common-ruby
```

Comment:
Maybe it is a good time to implement it? :wink:

Reply:
Yes, it is! Is this something we'd do or ask whoever maintains that repo 🤔 A lot of the other apps/services use `app-common-python` and that's being updated in a similar way

Reply:
Petr opened [a PR](https://github.com/RedHatInsights/clowder-common-ruby/pull/29) in the clowder-common-ruby gem with the new fields that are needed 🙂  So when it's merged and the version is updated we'll have `tlsPort` and `tlsCAPath` like `app-common-python` already has